### DISCUSSION TITLE
How can I specify dialect 🤔

### DISCUSSION THREAD

**Discussion by KeShengchuan** [reata/sqllineage#351] [General]
I see on the document that it is: 
`sqllineage -e "insert into analyze select * from foo;" --dialect=ansi`

But when I tested this code, it gave an error
**sqllineage: error: unrecognized arguments: --dialect=ansi**

So, how can I specify dialect ?

**Comment by KeShengchuan** [comment]
BTW: I'm using the latest 1.3.7 version
thanks in advance !

**Reply by reata** [reply]
Dialect is a new feature we introduce in v1.4.0. We will release it this week. Sorry for the confusion.

**Reply by KeShengchuan** [reply]
Thank you for your reply, I am really looking forward to the new features, I will continue to pay attention to the updates!
